src/azul/queues.py

```diff
@@ -74,9 +74,9 @@
     )
 
 
-@attrs.frozen(auto_attribs=True)
+@attrs.frozen(auto_attribs=True, kw_only=True)
```

Comment:
```suggestion
@attrs.frozen(kw_only=True)
```

Unless you're mixing annotated fields with unannotated ones, the default for `auto_attribs` is fine.